### ISSUE TITLE
Validation Loss issue

### ISSUE THREAD

**Issue by Hussain7252** [romarcg/traversability_estimation_pioneer#12]
While  trying to train the model I am facing an issue related to the validation loss. The validation loss is supposed to decrease as the model is trained and when it overfits the loss starts increasing. But in my case the validation loss in the first 5 epoch is decreasing then after, it keeps  on increasing and the validation accuracy is always  around  0.80 since 1st epoch until the 50th.

I did not understand this behavior. Can someone explain
